packages/utils/src/Path.ts

```diff
@@ -0,0 +1,34 @@
+import { Maybe } from './types';
```

Comment:
@ardatan As part of building for ESM you are trying to stick to "all imports have an extension", right? Looks like this line and the MaybePromise import in jsutils.ts are missing the extension, which breaks builds with tsc moduleResolution nodenext.

Reply:
PR on its way to fix and add lint rule.

Reply:
#4795